cmd/lncli/cmd_chain.go

```diff
@@ -0,0 +1,91 @@
+package main
```

Comment:
Since the `chainrpc` subserver is optional with a build tag, we also need to do the build tag thing here. That's why it might be easier to put these two new commands below a sub-command. So it would be `lncli chain getblock` instead of just `lncli getblock` for example. See `walletrpc_active.go` and `walletrpc_default.go`.